packages/app/obojobo-repository/shared/components/dashboard.jsx

```diff
@@ -647,7 +674,10 @@ function Dashboard(props) {
 				<div className="repository--item-list--collection">
 					<div className="repository--item-list--collection--item-wrapper">
 						<div className="repository--item-list--row">
-							<div className="repository--item-list--collection--item--multi-wrapper">
+							<div
+								className="repository--item-list--collection--item--multi-wrapper"
+								ref={newCollection}
```

Comment:
Setting the collections container as the ref instead of the new collection itself seems like it shouldn't work, but I can see that somehow it does. I'm willing to let it ride, but we may have to go back and adjust this to add the ref to a specific collection component as it's being generated inside `renderCollections`.